build/charts/antrea/templates/controller/clusterrole.yaml

```diff
@@ -290,3 +290,31 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - crd.antrea.io
+    resources:
+      - supportbundlecollections
+    verbs:
+      - get
+      - watch
+      - list
+  - apiGroups:
+      - crd.antrea.io
+    resources:
+      - supportbundlecollections/status
+    verbs:
+      - update
+---
+kind: ClusterRole
```

Comment:
Could we publish these YAMLs with Nephe as they are not needed by Antrea for K8s?

Reply:
In this way, the upstream user can not use this feature with on-prem VMs?

Reply:
My concern is about adding permissions for controller to read all Secrets. We should not do that for a regular K8s cluster deployment. For non-Nephe use cases, could we just document the role binding requirement?

Reply:
I remembered @reachjainrahul said that Nephe would not do much on the support bundle feature, so I doubt maybe it is not a better choice to place to the RBAC changes in Nephe.

If your concern is the impact on regular K8s cluster, I would prefer to move this special ClusterRole/RoleBinding out from the default antrea yaml, but place it to path `build/yamls/externalnode/`, just like vm-agent-rbac.yaml . What is your thought?

Reply:
Does it mean users (inc. Nephe) need to apply another YAML to use the feature? I am fine with that, but feel easier for Nephe to include it in its YAML.

Reply:
> Does it mean users (inc. Nephe) need to apply another YAML to use the feature? 

Yes. But if we only have it in Nephe but do not maintain it in antrea, the non-Nephe users can not use this feature.

Reply:
We can provide a YAML in Antrea too. My question is just should we include the role binding in the Nephe YAML to simplify Nephe deployment.

Reply:
In my mind, Nephe is not planned with support bundle collection feature. @reachjainrahul please correct me, and share your thought?